models/electric_relaxation/src/offline_evaluation/evaluate_model.py

```diff
@@ -34,7 +37,10 @@ def evaluate_model(model_config):
     """
     print("Evaluating...")
 
-    df_calib = pd.read_parquet(model_path/"data"/"generated"/"calibration_predictions.parquet") 
+    PATH_MODEL, PATH_RAW, PATH_PROCESSED, PATH_GENERATED = setup_data_paths(PATH)
```

Comment:
with the new implementation, you should not get PATH_MODEL from here. You can get it from setup_model_path but given the use below I think you should just use setup_artifacts_path